### ISSUE TITLE
ExpandingTextArea loses focus on Safari

### ISSUE THREAD

**Issue by tehapo** [vaadin/tori#76]
ExpandingTextArea loses focus on Safari everytime the area is expanded. This happens only when the application is deployed as a portlet inside Liferay portal.

The blue focus indicator ring is still left on the text area but the cursor is missing and text input is not possible.
